Evergreen/Apps/Get-BlueJ.ps1

```diff
@@ -0,0 +1,46 @@
+Function Get-BlueJ {
+    <#
+        .SYNOPSIS
+            Get the current version and download URIs for the supported releases of BlueJ.
+
+        .NOTES
+            Author: Andrew Cooper
+            Twitter: @adotcoop
+    #>
+    [OutputType([System.Management.Automation.PSObject])]
+    [CmdletBinding(SupportsShouldProcess = $False)]
+    param (
+        [Parameter(Mandatory = $False, Position = 0)]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSObject]
+        $res = (Get-FunctionResource -AppName ("$($MyInvocation.MyCommand)".Split("-"))[1]),
+
+        [Parameter(Mandatory = $False, Position = 1)]
+        [ValidateNotNull()]
+        [System.String] $Filter
+    )
+
+    # Query the BlueJ update API
+    $iwcParams = @{
+        Uri       = $res.Get.Uri
+        UserAgent = $res.Get.UserAgent
+    }
+
+    $Content = Invoke-WebRequestWrapper @iwcParams
+
+    If ($Null -ne $Content) {
+
+        # Convert from UTF8
+        $Updates = [System.Text.Encoding]::UTF8.GetString($Content)
```

Comment:
Could you wrap these in some error checking? Just in case the `GetString` fails for example.

```
# Convert from UTF8
$Updates = [System.Text.Encoding]::UTF8.GetString($Content)

# Latest version is stored at the top of the file on its own line
$LatestVersion = $Updates.Split([Environment]::NewLine)[0]
```

Reply:
I added a check on the GetString, and also a validation that the version found at the top of the file is actually in SemVer format - if not, then the creation of the download url would fail.